--- 1/po/cat-id-tbl.c
+++ 2/po/cat-id-tbl.c
@@ -39,32 +39,38 @@
   {"?? getopt returned character code 0%o ??\n", 30},
   {"non-option ARGV-elements: ", 31},
   {"memory exhausted", 32},
-  {"%s: warning: %s: %s\n", 33},
-  {"writing output", 34},
-  {"Binary file %s matches\n", 35},
-  {"(standard input)", 36},
-  {"Usage: %s [OPTION]... PATTERN [FILE]...\n", 37},
-  {"Try `%s --help' for more information.\n", 38},
-  {"Usage: %s [OPTION]... PATTERN [FILE] ...\n", 39},
+  {"writing output", 33},
+  {"Binary file %s matches\n", 34},
+  {"(standard input)", 35},
+  {"%s: warning: %s: %s\n", 36},
+  {"recursive directory loop", 37},
+  {"Usage: %s [OPTION]... PATTERN [FILE]...\n", 38},
+  {"Try `%s --help' for more information.\n", 39},
+  {"Usage: %s [OPTION]... PATTERN [FILE] ...\n", 40},
   {"\
 Search for PATTERN in each FILE or standard input.\n\
+Example: %s -i 'hello.*world' menu.h main.c\n\
 \n\
-Regexp selection and interpretation:\n\
+Regexp selection and interpretation:\n", 41},
+  {"\
   -E, --extended-regexp     PATTERN is an extended regular expression\n\
-  -F, --fixed-regexp        PATTERN is a fixed string separated by newlines\n\
-  -G, --basic-regexp        PATTERN is a basic regular expression\n\
+  -F, --fixed-strings       PATTERN is a set of newline-separated strings\n\
+  -G, --basic-regexp        PATTERN is a basic regular expression\n", 42},
+  {"\
   -e, --regexp=PATTERN      use PATTERN as a regular expression\n\
   -f, --file=FILE           obtain PATTERN from FILE\n\
   -i, --ignore-case         ignore case distinctions\n\
   -w, --word-regexp         force PATTERN to match only whole words\n\
-  -x, --line-regexp         force PATTERN to match only whole lines\n", 40},
+  -x, --line-regexp         force PATTERN to match only whole lines\n\
+  -z, --null-data           a data line ends in 0 byte, not newline\n", 43},
   {"\
 \n\
 Miscellaneous:\n\
   -s, --no-messages         suppress error messages\n\
-  -v, --revert-match        select non-matching lines\n\
+  -v, --invert-match        select non-matching lines\n\
   -V, --version             print version information and exit\n\
-      --help                display this help and exit\n", 41},
+      --help                display this help and exit\n\
+      --mmap                use memory-mapped input if possible\n", 44},
   {"\
 \n\
 Output control:\n\
@@ -79,37 +85,35 @@
   -r, --recursive           equivalent to --directories=recurse.\n\
   -L, --files-without-match only print FILE names containing no match\n\
   -l, --files-with-matches  only print FILE names containing matches\n\
-  -c, --count               only print a count of matching lines per FILE\n", 42},
+  -c, --count               only print a count of matching lines per FILE\n\
+  -Z, --null                print 0 byte after FILE name\n", 45},
   {"\
 \n\
 Context control:\n\
   -B, --before-context=NUM  print NUM lines of leading context\n\
   -A, --after-context=NUM   print NUM lines of trailing context\n\
   -C, --context[=NUM]       print NUM (default 2) lines of output context\n\
-                            unless overriden by -A or -B\n\
-  -NUM                      same as --contex=NUM\n\
+                            unless overridden by -A or -B\n\
+  -NUM                      same as --context=NUM\n\
   -U, --binary              do not strip CR characters at EOL (MSDOS)\n\
   -u, --unix-byte-offsets   report offsets as if CRs were not there (MSDOS)\n\
 \n\
-If no -[GEF], then `egrep' assumes -E, `fgrep' -F, else -G.\n\
-With no FILE, or when FILE is -, read standard input. If less than\n\
-two FILEs given, assume -h. Exit with 0 if matches, with 1 if none.\n\
-Exit with 2 if syntax errors or system errors.\n", 43},
+`egrep' means `grep -E'.  `fgrep' means `grep -F'.\n\
+With no FILE, or when FILE is -, read standard input.  If less than\n\
+two FILEs given, assume -h.  Exit status is 0 if match, 1 if no match,\n\
+and 2 if trouble.\n", 46},
   {"\
 \n\
-Report bugs to <bug-gnu-utils@gnu.org>.\n", 44},
-  {"invalid context length argument", 45},
-  {"infalid context length argument", 46},
-  {"you may specify only one of -E, -F, or -G", 47},
-  {"matcher already specified", 48},
-  {"unknown directories method", 49},
-  {"grep (GNU grep) %s\n", 50},
-  {"\
-Copyright (C) 1988, 92, 93, 94, 95, 96, 97 Free Software Foundation, Inc.\n", 51},
+Report bugs to <bug-gnu-utils@gnu.org>.\n", 47},
+  {"conflicting matchers specified", 48},
+  {"invalid context length argument", 49},
+  {"unknown directories method", 50},
+  {"%s (GNU grep) %s\n", 51},
+  {"Copyright (C) 1988, 1992-1998, 1999 Free Software Foundation, Inc.\n", 52},
   {"\
 This is free software; see the source for copying conditions. There is NO\n\
-warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.\n", 52},
-  {"memory exhausted\n", 53},
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.\n", 53},
+  {"memory exhausted\n", 54},
 };
 
-int _msg_tbl_length = 53;
+int _msg_tbl_length = 54;
--- 1/THANKS
+++ 2/THANKS
@@ -1,6 +1,6 @@
 Aharon Robbins        <arnold@gnu.org>
 Alain Magloire        <alainm@rcsm.ece.mcgill.ca>
-Andreas Schwab        <schwab@issan.informatik.uni-dortmund.de>
+Andreas Schwab        <schwab@suse.de>
 Andreas Ley           <andy@rz.uni-karlsruhe.de>
 Ben Elliston          <bje@cygnus.com>
 David J MacKenzie     <djm@catapult.va.pubnix.com>
